addons/html_editor/static/src/editor/core/delete_plugin.js

```diff
@@ -441,907 +556,168 @@ export class DeletePlugin extends Plugin {
             }
         }
         // @todo: this assumes the common ancestor does not change. Double check this.
-        return { ...selection, startContainer, startOffset, endContainer, endOffset };
+        return { startContainer, startOffset, endContainer, endOffset };
```

Comment:
and return a `selection` object, so we are not confused later when manipulating something